examples/performance-subscribe/AppManifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "manifestVersion": "v3",
+    "name": "performance-subscribe",
+    "interfaces": [
+        {
+            "type": "vehicle-signal-interface",
+            "config": {
+                "src": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
```

Comment:
should be 3.1.1

Reply:
Or 4.0, 4.1 - whatever. The signal catalog isn't relevant here - the app is not using any model.

Basically, we would need a option to tell: "I need access to a data broker with interface version x.y.z"

Reply:
Updated it to VSS 4.0 as this is referred by the list of signals

Reply:
yeah fine for me as it is our new standard